src/decision/location_selection.jl

```diff
@@ -281,26 +299,26 @@ Selection score
 """
 function selection_score(
     ranks::YAXArray{T,3},
-    iv_type::Union{Symbol,Int64},
-)::YAXArray where {T<:Union{Int64, Float32, Float64}}
+    iv_type::Union{Symbol,Int64}
+)::YAXArray where {T<:Union{Int64,Float32,Float64}}
```

Comment:
funny that it wants comma-space elsewhere, but not with types.

Reply:
This is a separate rule https://domluna.github.io/JuliaFormatter.jl/stable/#whitespace_typedefs